accepted/2020/better-obsoletion/obsoletions-in-net5.md

```diff
@@ -0,0 +1,236 @@
+            * `System.Security.Permissions.EnvironmentPermission`
+            * `System.Security.Permissions.FileDialogPermission`
+            * `System.Security.Permissions.FileIOPermission`
+            * `System.Security.Permissions.GacIdentityPermission`
+            * `System.Security.Permissions.IsolatedStoragePermission`
+                * `System.Security.Permissions.IsolatedStorageFilePermission`
+            * `System.Security.Permissions.KeyContainerPermission`
+            * `System.Security.Permissions.MediaPermission`
+            * `System.Security.Permissions.PublisherIdentityPermission`
+            * `System.Security.Permissions.ReflectionPermission`
+            * `System.Security.Permissions.RegistryPermission`
+            * `System.Security.Permissions.ResourcePermissionBase`
+                * `System.Diagnostics.EventLogPermission`
+                * `System.Diagnostics.PerformanceCounterPermission`
+                * `System.DirectoryServices.DirectoryServicesPermission`
+                * `System.ServiceProcess.ServiceControllerPermission`
+            * `System.Security.Permissions.SecurityPermission`
+            * `System.Security.Permissions.SiteIdentityPermission`
+            * `System.Security.Permissions.StorePermission`
+            * `System.Security.Permissions.StrongNameIdentityPermission`
+            * `System.Security.Permissions.TypeDescriptorPermission`
+            * `System.Security.Permissions.UIPermission`
+            * `System.Security.Permissions.UrlIdentityPermission`
+            * `System.Security.Permissions.WebBrowserPermission`
+            * `System.Security.Permissions.ZoneIdentityPermission`
+            * `System.Transactions.DistributedTransactionPermission`
+            * `System.Web.AspNetHostingPermission`
+            * `System.Xaml.Permissions.XamlLoadPermission`
+        * Enums: (everything from the `System.Security.Permissions` namespace *except* `PermissionState`, which is used by `PrincipalPermission`; see below)
+            * `DataProtectionPermissionFlags`
+            * `EnvironmentPermissionAccess`
+            * `FileDialogPermissionAccess`
+            * `FileIOPermissionAccess`
+            * `HostProtectionResource`
+            * `IsolatedStorageContainment`
+            * `KeyContainerPermissionFlags`
+            * `MediaPermissionAudio`
+            * `MediaPermissionImage`
+            * `MediaPermissionVideo`
+            * `ReflectionPermissionFlag`
+            * `RegistryPermissionAccess`
+            * `SecurityAction`
+            * `SecurityPermissionFlag`
+            * `StorePermissionFlags`
+            * `TypeDescriptorPermissionFlags`
+            * `UIPermissionClipboard`
+            * `UIPermissionWindow`
+            * `WebBrowserPermissionLevel`
+1. PrincipalPermission is no longer supported
+    * DiagnosticId: `MSLIB0012`
+    * APIs:
+        * `System.Security.Permissions.PrincipalPermission`
```

Comment:
@bartonjs Could you please comment on https://github.com/dotnet/designs/pull/139#discussion_r453846088 ?

I do not see why `PrincipalPermission` should be its own bucket separate from all other permissions.

Reply:
I had combined it in but then ended up having to keep the `PrincipalPermissionAttribute` constructor as its own ID since it's an **error** level and the others are **warnings**. But `PrincipalPermission` falls into the general CAS category.